.github/actions/start-opensearch/action.yml

```diff
@@ -0,0 +1,199 @@
+name: 'Start OpenSearch'
+description: 'Downloads latest build of OpenSearch, installs a plugin, executes and script and then starts OpenSearch on localhost:9200'
+
+inputs:
+  platform:
+    description: 'What platform is this action running on, e.g. "windows-latest" ? '
+    required: true
+
+  opensearch-version:
+    description: 'The version of OpenSearch that should be used, e.g "3.0.0"'
+    required: true
+
+  plugin-name:
+    description: 'The the name of the plugin to use, such as security-dashboards-plugin'
+    required: true
+
+  plugin-start-script:
+    description: 'The file name for the configuration script for the plugin such as install_demo_configurations'
+    required: true
+  # plugins/opensearch-security/tools/install_demo_configuration.sh
+
+  java-version:
+    description: 'The java version to be used, must be either 11 or 17'
+    required: true
+
+  plugin-zip:
```

Comment:
What do you think of making `plugin-zip` is the same as `plugin-name` so this parameter can be removed?

Reply:
That sounds good but right now the files on docker-host have different naming conventions so that would have to be changed first.

Reply:
Seems like the docker start-up script is where this is happening, is there more to it?

Reply:
I am not sure what the docker startup script is--the first mention of anything docker related is at that line in the file.